web/src/components/tasks/business-formation/contacts/RegisteredAgent.test.tsx

```diff
@@ -45,6 +45,7 @@ describe("Formation - Registered Agent Field", () => {
   });
 
   it("auto-fills fields when agent is manual entry", async () => {
+    const randomAgentOfficeAddressCity = `agent-city-${randomInt()}`;
```

Comment:
pebble - in the other staget data example, we are not using the `randomInt` func, why use it? If your using it, why should we selectively use it?

Reply:
it seems like this pattern is used in other test.. we should be consistent.